Tests/test_SeqIO.py

```diff
@@ -131,11 +131,7 @@ class TestZipped(unittest.TestCase):
 
     def test_gzip_fastq(self):
         """Testing FASTQ with gzip."""
-        if sys.version_info >= (3,):
-            mode = "rt"
-        else:
-            # Workaround for bug https://bugs.python.org/issue30012
-            mode = "r"  # implicitly text mode, rejects making explicit
+        mode = "rt"
         with gzip.open("Quality/example.fastq.gz", mode) as handle:
             self.assertEqual(3, len(list(SeqIO.parse(handle, "fastq"))))
         if 3 <= sys.version_info[0]:
```

Comment:
Since you are tweaking the lines immediately above, might as well remove this if statement since we now require at least Python 3?

Reply:
On the other hand, there are quite a few of these to include...

Reply:
I removed them, but is that actually correct? `3 <= sys.version_info[0]` is actually `True` in python 3- is the test intended for python 3 only in fact?

Reply:
As per my comment below, I meant make the blocks of code no longer conditional - they should always run now that we've dropped Python 2 support. Apologies for the confusion.